TechnicoBackend/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.EntityFrameworkCore;
+using TechnicoBackend.Data;
+using TechnicoBackend.Repositories;
+using TechnicoBackend.Services;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddDbContext<TechnicoDbContext>(options =>
+    options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection")));
+
+builder.Services.AddScoped<UserRepository>();
```

Comment:
Consider using interfaces and connecting them to the respected Services and Repositories. 
e.g. builder.Services.AddScoped<IUserRepository, UserRepository>();